src/FbgemmI8Depthwise3DAvx2.cc

```diff
@@ -9,6 +9,7 @@
 
 #include <string>
 #include <tuple> // for tie
+#include <stdexcept> // for logic_error
```

Comment:
nit: can you move stdexcept before string? In general we should run clang formatter before checking in the code. In VSCode you can easily run the formatter.